wazuh/wazuh-kibana-app
# Fix repeated request in groups-table

### Description
Fixes repeated requests in the group table when adding a group or refreshing the table
 
### Issues Resolved
- #5241

### Evidence
<details><summary>Details</summary>

Refresh

![image](https://github.com/wazuh/wazuh-kibana-app/assets/63758389/e24f9338-6512-49dd-aea9-cab29491a360)

Add new group

![image](https://github.com/wazuh/wazuh-kibana-app/assets/63758389/751b15d7-607e-48bc-9042-31716edaaaeb)

![image](https://github.com/wazuh/wazuh-kibana-app/assets/63758389/93fc4c5a-413d-4111-884f-f277bcb1594c)

</details> 

### Test

1. Navigate to `Management/groups`
2. Click on `Refresh | Add new group`
3. See requests in the console

Legend:
:black_circle:: none
:green_circle:: pass
:yellow_circle:: warning
:red_circle:: fail
:white_circle:: not applicable

## UI

| Test | Chrome | Firefox | Safari |
| --- |  --- | --- | --- |
| Requests in `Refresh` | :black_circle: | :black_circle: | :black_circle: |
| Requests in `Add new group` | :black_circle: | :black_circle: | :black_circle: |

**Details**
<details>
<summary>:black_circle: Requests in `Refresh`</summary>

Chrome - :black_circle:

Firefox - :black_circle:

Safari - :black_circle:

</details>

<details>
<summary>:black_circle: Requests in `Add new group`</summary>

Chrome - :black_circle:

Firefox - :black_circle:

Safari - :black_circle:

</details>




### Check List
